### ISSUE TITLE
No news downloader in plugins menu

### ISSUE THREAD

**Issue by mwoz123** [koreader/koreader#6001]
* KOReader version: last nighty 
* Device: android

#### Issue

Although "news downloader" is enabled in "plugin management" it's neither shown in plugins menu nor in "more plugins" 

#### Steps to reproduce

Install last nightly build open plugins menu - > there is no news downloader

**Comment by Frenzie** [comment]
I can't confirm, and nothing should have changed. Could you attach the startup log?

**Comment by mwoz123** [comment]
It's on Inkbook Prime. 

Previusly I also had no ND - I think I had then version from pazos dropbox (https://github.com/koreader/koreader/issues/5787#issuecomment-579769702)
or one of January or February build.

After upgrade (to today recent build) it's still not in place.

BTW is there a way to get Koreader android logs without adb ? Cause I seen a entry in menu about debug logs

**Comment by Frenzie** [comment]
If you're talking about developer options, that only affects verbosity. ADB is required for the logs, see the new issue template.

**Comment by mwoz123** [comment]
After I enabled debug options and re-enabled ND from menu (prev for test I disabled it) now clicking at top menu crashes my Koreader. Log attached:


[k2.log](https://github.com/koreader/koreader/files/4382035/k2.log)

Maybe cleaning Koreader android caches will fix it- I'll try.

**Comment by Frenzie** [comment]
It looks like it's erroring while copying a file.

```

I/KOReader( 3980): }
I/KOReader( 3980):  RD loaded plugin systemstat at plugins/systemstat.koplugin
E/KOReader( 3980): [NativeThread]: failed to run script: ffi/util.lua:164: attempt to index local 'tfp' (a nil value)
E/KOReader( 3980): [NativeThread]: Stopping due to previous errors
```

Which doesn't really happen anywhere without specific user request, except apparently in NewsDownloader initialization:

https://github.com/koreader/koreader/blob/0599c440cc934c1dceb5a2fed4aebb02bf01823e/plugins/newsdownloader.koplugin/main.lua#L154-L156

Stupid questions first, do you have enough space left and is it writable?

**Comment by mwoz123** [comment]
Android memory settings says I have 607mb free internal mem (root? Ram?whatever :p) and 3.81 GB nand flash free.

It's seems be writable as I was able to create new dirs there. 

Right you gave me idea. Some time ago inkBOOK dropped support for their ownCloud. So I moved all of my books folder from 'drive/mwoz123/...' to 'books'. 

I should have done this first from koreader news downloader cause now it gets crashed cause it can't find the previous news downloader config path (this is nil)! 

So my problem is solved. Thanks a lot. 

Do you want me to create a bug to create ad hoc news downloader config file when current folder is missing (I think we might be doing this only once in some init set up) or it's won't happen too often and not needed? 

Once again many thanks for help :)

**Comment by Frenzie** [comment]
Yeah, it's better to track that. :+1:

**Comment by mwoz123** [comment]
Installed koreader on my phone, tried to reproduce but I found out that it doesn't work when the config was moved/removed from 'koreader/news' not from any other custom set news downloader folder. But I'll create a bug anyway.
